src/containers/apps/AppsTable.tsx

```diff
@@ -80,6 +81,26 @@ class AppsTable extends Component<
                 dataIndex: 'instanceCount',
                 key: 'instanceCount',
                 align: ALIGN,
+                render: (instanceCount: number, row, index) => {
+                    return (
+                        <Button
+                            style={{ margin: -5 }}
+                            onClick={async () => {
+                                const warning =
+                                    'Do you want to turn this app OFF?'
+                                if (instanceCount && !window.confirm(warning))
```

Comment:
1- I think we should also have a confirmation for `turning on` to avoid accidental clicks.

2- Suggest using `modal.confirm` from ant design to match the existing UI.